archunit/src/main/java/com/tngtech/archunit/library/GeneralCodingRules.java

```diff
@@ -15,10 +15,6 @@
  */
 package com.tngtech.archunit.library;
 
-import java.util.Collection;
```

Comment:
Could you please use the formatter that's provided in the repo? -> https://github.com/TNG/ArchUnit/tree/main/develop
Just to keep unrelated fluff low and avoid formatting back and forth :slightly_smiling_face: